ft-email-auth-provider/src/handlers/create_account.rs

```diff
@@ -7,6 +93,9 @@ struct CreateAccount {
     email_confirmation_code: String,
     user_id: Option<ft_sdk::UserId>,
     email_confirmation_sent_at: chrono::DateTime<chrono::Utc>,
+    /// do not send a confirmation email or set a confirmation key in db if the user is
+    /// `pre_verified`
```

Comment:
```suggestion
    /// do not send a confirmation email or set a confirmation key in db if the user is
    /// `pre_verified`. This can be set by apps like subscription app.
```